healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/service/SofaBootReadinessCheckEndpoint.java

```diff
@@ -35,7 +36,7 @@
  * @author liangen
  * @version $Id: StartUpHealthCheckStatusCheckInfo.java, v 0.1 2018年02月02日 下午11:34 liangen Exp $
  */
-
+@ConfigurationProperties(prefix = "endpoints.sofaboot.health.readiness")
```

Comment:
`ConfigurationProperties` 的正确用法：[参考这里](https://docs.spring.io/spring-boot/docs/1.4.2.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-validation)

> If you define a set of configuration keys for your own components, we recommend you group them in a POJO annotated with @ConfigurationProperties. You should also be aware that, since @Value does not support relaxed binding, it is not a good candidate if you need to provide the value by using environment variables.

Reply:
这里正确的用法是啥意思呀？ 另外现在前缀是要改成 `com.alipay.sofa.healthcheck.readiness` 吗